### ISSUE TITLE
[Crash]: Changing interface size while chat is visible causes segfault

### ISSUE THREAD

**Issue by MiguelHorta** [OpenTTD/OpenTTD#12755]
### Version of OpenTTD

14.1 (Linux), 14.1 (windows)

### Expected result

No crash

### Actual result

Crash.

Bt:
```#0  0x00007ffff517b401 in ??? () at /usr/lib/libc.so.6
#1  0x000055555644d538 in Blitter_40bppAnim::CopyFromBuffer (this=0x555558d7b290, video=0x7fff9dafd028, src=0x555559335400, width=1024, height=51) at /Extra/repos/openttd/OpenttdUpstream/src/blitter/40bpp_anim.cpp:422
#2  0x0000555556533571 in NetworkUndrawChatMessage () at /Extra/repos/openttd/OpenttdUpstream/src/network/network_chat_gui.cpp:158
#3  0x000055555692f320 in RedrawScreenRect (left=0, top=0, right=2560, bottom=1386) at /Extra/repos/openttd/OpenttdUpstream/src/gfx.cpp:1402
#4  0x000055555692f516 in DrawDirtyBlocks () at /Extra/repos/openttd/OpenttdUpstream/src/gfx.cpp:1479
#5  0x0000555556ddd648 in UpdateWindows () at /Extra/repos/openttd/OpenttdUpstream/src/window.cpp:3070
#6  0x0000555556817154 in VideoDriver::Tick (this=0x555557e3ea40) at /Extra/repos/openttd/OpenttdUpstream/src/video/video_driver.cpp:151
#7  0x000055555681042d in VideoDriver_SDL_Base::LoopOnce (this=0x555557e3ea40) at /Extra/repos/openttd/OpenttdUpstream/src/video/sdl2_v.cpp:633
#8  0x0000555556810464 in VideoDriver_SDL_Base::MainLoop (this=0x555557e3ea40) at /Extra/repos/openttd/OpenttdUpstream/src/video/sdl2_v.cpp:651
#9  0x0000555556b254b7 in openttd_main (argc=1, argv=0x7fffffffd958) at /Extra/repos/openttd/OpenttdUpstream/src/openttd.cpp:829
#10 0x00005555561d40b7 in main (argc=1, argv=0x7fffffffd958) at /Extra/repos/openttd/OpenttdUpstream/src/os/unix/unix_main.cpp:32
```

### Steps to reproduce

1. Join any network game
2. Make sure chat is visible
3. Change interface size from within game options widow
4. Crash

**Comment by PeterN** [comment]
Already fixed by #12705

**Comment by MiguelHorta** [comment]
Duh.. I did search within the reported issues, but I forgot to check the commits. Well, great to know it is fixed. 👍
